lib/components/molecules/List/List.js

```diff
@@ -21,9 +21,23 @@ type Props = {
   borderSeparator?: boolean,
   noMargin?: boolean,
   inheritedStyles?: string,
+  noDefaultView?: string | Boolean,
+  initialSelectedIndex?: Number | Boolean,
```

Comment:
Whats the use of initialSelectedIndex on list?

Reply:
all changes have been roll backed